modules/LemDCBM400600/main/lem_dcbm_time_sync_helper.hpp

```diff
@@ -46,6 +46,8 @@ class LemDCBMTimeSyncHelper {
         timing_constants(tc), ntp_spec(std::move(ntp_spec)), unsafe_period_start_time({}) {
     }
 
+    virtual ~LemDCBMTimeSyncHelper() = default;
+
```

Comment:
Is this needed to **sucessfully** run the unit tests?

Reply:
Correct